tests/Aspire.Hosting.Testing.Tests/TestingFactoryTests.cs

```diff
@@ -48,3 +48,12 @@ private sealed record WeatherForecast(DateOnly Date, int TemperatureC, string? S
         public int TemperatureF => 32 + (int)(TemperatureC / 0.5556);
     }
 }
+
+public class DistributedApplicationFixtureWithAllowUnsecureTransport<TEntryPoint> : DistributedApplicationFixture<TEntryPoint> where TEntryPoint: class
+{
+    protected override void OnBuilding(DistributedApplicationBuilder applicationBuilder)
+    {
+        applicationBuilder.Configuration[KnownConfigNames.AllowUnsecuredTransport] = "true";
```

Comment:
This isn't needed right?

Reply:
Correct. I'll remove this before merging.